src/aggregate/aggregate.h

```diff
@@ -261,6 +261,10 @@ typedef struct AREQ {
 
   // The offset of the prefixes in the command
   size_t prefixesOffset;
+
+  // Indicates whether the query has timed out.
+  // Useful for query with cursor and RETURN policy
+  bool has_timedout;
```

Comment:
Today, we have this in the error structure of the request. Can't we reuse it instead?